src/Functions/Public/content-management-service/Import-vRAContentData.ps1

```diff
@@ -0,0 +1,79 @@
+﻿function Import-vRAContentData {
+<#
+    .SYNOPSIS
+    Import a yaml file associated with vRA content
+
+    .DESCRIPTION
+    Import a yaml file associated with vRA content
+
+    .PARAMETER ContentType
+    The Content Type of the imported item such as composite-blueprint or property-group
+
+    .PARAMETER Path
+    The path to file to import
+
+    .INPUTS
+    System.String
+
+    .OUTPUTS
+    System.String
+
+    .EXAMPLE
+    Import-vRAContentData -Path ./CentOS.yaml -ContentType Composite-Blueprint
```

Comment:
Change
```Composite-Blueprint```
to
```composite-blueprint```

vRA doesn't like the uppercase letters

Reply:
done